language-design.md

```diff
@@ -17,7 +17,56 @@ For each quote, describe what it was about the quote that led you pick it.
 
 **Response**
 
-
+> It is one thing to have learned the syntax of a language and another to have 
+> grasped the underlying paradigm [Yang and Radkin, 2015]
+
+We agree with this quote from _C is Manly, Python is for "n00bs": How False
+Stereotypes Turn Into Technical "Truths"_ for the most part, but in the case
+of Python and C, it is more complicated. Python does not require the user to
+have as sophisticated of an understanding of underlying paradigms because
+the intent of Python is to be user-friendly. To use C, on the other hand,
+it is necessary for the user to have at least a basic understanding of 
+underlying principles. An example is passing by reference. In C, you must
+explicitly pass a reference to a variable, whereas in Python you simply
+pass a parameter. 
+
+
+> ```
+> integer i = 0
+> repeat 10 times
+> i = i + 1
+> end 
+> ```
+> That still looks mostly like Greek to me, but Stefik compares it to this 
+> equivalent statement in Java ("which is similar to Perl is some ways," he 
+> says):
+> ```
+> for(int i = 0; i < 10; i++) {
+> } 
+> ```
+> That’s not Greek, it’s Klingon 
+[Pavlus, 2012] 
+
+To us as programmers, the second example of Java code is far clearer. The 
+more English-y syntax of the Quorum example may be clearer to someone
+unfamiliar with programming. The Java code is a form of shorthand for the
+Quorum, and if you use a phrase enough times shorthand becomes quicker
+and easier to understand. 
+
+> A cathedral is a huge church. It may be made of stone; it may fill you with
+> awe; but the key thought, in the mind of Eric Raymond, is that there is but
+> one design, one grand plan, which may take a long time—many years—to make 
+> real
+[Steele, 1998]
+
+This was a very appropriate metaphor for beginning to learn programming. 
+Most people would learn a general concept language first, like Java, and
+the amount it can do could be overwhelming. A smaller language, like Scratch,
+might provide a gentler introduction to programming in general. When you're
+first learning and being taught conventions, it throws back to the original
+grand plan for how the language should be used. As you learn a language 
+more, you can see that it's still growing and that it's not necessarily
+a cathedral. 
```

Comment:
I thought this was a great metaphor too.  Especially when I first began programming,  I never thought of mature languages as still in development because they really did feel like cathedrals.